sample-templates/query-assist-agent-claude.json

```diff
@@ -1,90 +1,108 @@
 {
   "name": "Query Assist Agent",
-  "description": "Create a Query Assist Agent using Bedrock and Sagemaker models",
+  "description": "Create a Query Assist Agent using Claude on BedRock",
   "use_case": "REGISTER_AGENT",
   "version": {
     "template": "1.0.0",
-    "compatibility": [
-      "2.12.0",
-      "3.0.0"
-    ]
+    "compatibility": ["2.13.0", "3.0.0"]
   },
   "workflows": {
     "provision": {
+      "user_params": {},
       "nodes": [
         {
-          "id": "create_openai_connector",
+          "id": "create_claude_connector",
           "type": "create_connector",
+          "previous_node_inputs": {},
           "user_inputs": {
-            "name": "OpenAI Chat Connector",
-            "description": "The connector to public OpenAI model service for GPT 3.5",
             "version": "1",
-            "protocol": "http",
-            "parameters": {
-              "endpoint": "api.openai.com",
-              "model": "gpt-3.5-turbo"
-            },
-            "credential": {
-              "openAI_key": "PUT_YOUR_API_KEY_HERE"
-            },
+            "name": "Claude instant runtime Connector",
+            "protocol": "aws_sigv4",
+            "description": "The connector to BedRock service for Claude model",
             "actions": [
               {
-                "action_type": "predict",
+                "headers": {
+                  "x-amz-content-sha256": "required",
+                  "content-type": "application/json"
+                },
                 "method": "POST",
-                "url": "https://${parameters.endpoint}/v1/chat/completions"
+                "request_body": "{\"prompt\":\"${parameters.prompt}\", \"max_tokens_to_sample\":${parameters.max_tokens_to_sample}, \"temperature\":${parameters.temperature},  \"anthropic_version\":\"${parameters.anthropic_version}\" }",
+                "action_type": "predict",
+                "url": "https://bedrock-runtime.us-west-2.amazonaws.com/model/anthropic.claude-instant-v1/invoke"
               }
-            ]
+            ],
+            "credential": {
+                "access_key": "<put your access key here>",
```

Comment:
we should need session_token also

Reply:
point me to the docs for that?

Reply:
https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/bedrock_connector_anthropic_claude_blueprint.md#2-create-connector-for-amazon-bedrock

Reply:
done